### ISSUE TITLE
consider TAP 8 for "Add/remove maintainers to a project" user story

### ISSUE THREAD

**Issue by lukpueh** [jku/repository-playground#12]
The user stories doc correctly points out that PEP480 does not have a scalable solution for project maintainer changes (see [here](https://github.com/lukpueh/repository-playground/blame/2ea5d8a71caf87545d1d454c62ad9e53abf9f0e4/docs/USER-STORIES.md#L52) and [here](https://github.com/lukpueh/repository-playground/blame/2ea5d8a71caf87545d1d454c62ad9e53abf9f0e4/docs/USER-STORIES.md#L76-L78)). 

[TAP 8](https://github.com/theupdateframework/taps/blob/master/tap8.md) seems like a remedy for this problem (most notably see [*Self-maintaining project role*](https://github.com/theupdateframework/taps/blob/master/tap8.md#self-maintaining-project-role)). This consideration could be mentioned in the user stories document, or just be tracked here in the issue.

**Comment by jku** [comment]
Yes, this is a good point. The metadata layout in https://github.com/jku/repository-playground/issues/4#issuecomment-1124932300 "solves" the issue with just plain TUF (using an additional project key layer) but it's definitely a bit unusual -- and almost doubles the number of roles in the system which is pretty bad for current snapshot setup.

The downside of TAP-8 is that it of course makes the TUF specification and implementation even more complex -- and the implementation doesn't exist.
